app/services/idv/steps/send_link_step.rb

```diff
@@ -65,9 +65,18 @@ def send_link
           to: formatted_destination_phone,
           link: link(session_uuid),
           country_code: Phonelib.parse(formatted_destination_phone).country,
+          service_provider: sp_name,
         )
       end
 
+      def service_provider
+        current_user&.identities&.order('created_at DESC')&.first
+      end
+
+      def sp_name
+        service_provider ? service_provider&.friendly_name : APP_NAME
```

Comment:
We should be able to access `current_sp` here ([source](https://github.com/18F/identity-idp/blob/69a3016076d6c17bbac8b81b0aed5b9f32e6710d/app/services/flow/base_step.rb#L79-L80)), as an alternative to implementing the other new helper method.

```suggestion
        current_sp&.friendly_name.presence || APP_NAME
```